deployments/dlb_plugin/base/intel-dlb-plugin.yaml

```diff
@@ -32,6 +32,13 @@ spec:
           readOnlyRootFilesystem: true
           allowPrivilegeEscalation: false
         terminationMessagePath: /tmp/termination-log          
+        resources:
+          requests:
+            memory: "50Mi"
+            cpu: 1
```

Comment:
How did you come up with "cpu: 1" ?

Reserving one CPU core for a plugin seems like much..

Reply:
And also what's the source for the `50Mi` value, same one for _all_ plugins?

Were all of the plugins' memory usage profiled (as it should)?

Reply:
From nowhere..! I just wanted to ask your opinions! :)

Reply:
kube-scheduler and others set cpu request to 100m or 250m without limits. As @eero-t said, memory amount should be profiled to get a request and limit values.

Reply:
Could you teach me how to profile them?
Let me do and update.!

Reply:
@hj-johannes-lee Run worst-case use-case for the given plugin [1] and measure CPU & RSS consumption just from `top`, and e.g. double those numbers for the resource requests.

[1] For GPU plugin, worst case would be running it with all features (resource management, monitoring etc) enabled on a node with maximum [2] number of devices, e.g. by [using](https://github.com/intel/intel-device-plugins-for-kubernetes/pull/1118) [gpu_fakedev](https://github.com/intel/intel-device-plugins-for-kubernetes/tree/main/cmd/gpu_fakedev).

[2] Kernel device numbering supports 128 DRI devices, e.g. 8 PFs with 15 VFs each.

Reply:
@eero-t Thanks..!
Before going further, let me ask opinions once more.
In case of QAT plugin, I measured using top command like this:

%CPU 3.3-3.9 (avg: 3.38) %MEM: 0.0
How would you conclude if you set the requests/limits?

Reply:
K8s memory limits/requests are not relative to host memory amount, but absolute memory usage values.  Check the "RES" column from "top" (reported as KB).

If the values are really small, I'd round them up quite a bit.  E.g. ~40m at least to 100m (0.1 CPU cores).